### ISSUE TITLE
error: library handler failed

### ISSUE THREAD

**Issue by adelina2kx** [MoacirSchmidt/dart_mssql#8]
This can’t be used normally. I’m on a win10 system. I installed all versions of Microsoft Visual C++ Redistributable. I also installed the OLDDB driver. I also copied dart_mssql_64.dll to the project directory, but the startup still prompts "error: library handler failed."

**Comment by MoacirSchmidt** [comment]
Did you "Rename to "dart_mssql.dll" according to your needs and copy it to the main directory of your project."?

**Comment by adelina2kx** [comment]
> Did you "Rename to "dart_mssql.dll" according to your needs and copy it to the main directory of your project."?

Yes, my dart and operating system are both 64-bit. I chose the 64-bit dll file and renamed it to "dart_mssql.dll" and moved it into the main directory of the project。

**Comment by thobiassilva** [comment]
Estou com o mesmo problema com o mesmo ambiente

**Comment by atreeon** [comment]
I'm getting this error too.

Just to confirm, I have copied the dart_mssql.dll to the root directory of my dart project (same folder as pubspec.yaml).  I've also installed oledb using this link

https://docs.microsoft.com/en-us/sql/connect/oledb/download-oledb-driver-for-sql-server?view=sql-server-ver15
